packages/authentication/modules/user/useUpdateUser/__tests__/useUpdateUser.test.tsx

```
@@ -13,9 +14,8 @@ import useUpdateUser from '..'
 import type { User } from '../../../../types/user'
 import request from './fixtures/request.json'
 
-interface UseUpdateUserReturn<TUser> extends Omit<UseMutationResult<TUser, unknown>, 'data'> {
-  user?: TUser
-}
+// @ts-ignore TODO: (BA-1081) investigate AxiosRequestHeaders error
+export const axiosMock = new MockAdapter(axios)
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)